src/components/index.js

```diff
@@ -60,3 +62,5 @@ export { Dropdown, DropdownList, DropdownListItem } from './Dropdown';
 export { PopperTooltip, CssTooltip } from './TooltipNew';
 
 export { Loader } from './Loader';
+
+export { Divider } from './Divider';
```

Comment:
Please add typescript types definitions if it's exported component

Reply:
But it hasn't any prop types

Reply:
so you need to export a ComponentType with empty props :P

Reply:
ada5c0d4ab1a71b4d2fe5edbf16b8421399e55dd